pkg/Library/Mobilenet/Classifier.js

```diff
@@ -29,33 +30,15 @@ hasChanges({imageRef, model}, state) {
 
 async perceive({imageRef, model}, service) {
   const args = {
-    kind: 'TensorFlowService', //'TfjsService',
+    kind: 'TensorFlowService',
     msg: 'toolClassify',
     data: {
       imageRef,
       modelUrl: model?.url,
       modelKind: 'mobilenet'
     }
   };
-  /////////////////
-  // HACKALERT!
-  // TODO(mariakleiner): actually run the service, instead of using hardcoded results.
-  // const classifierResults = await service(args);
-  const classifierResults = [
-    {
-      displayName: "strawberry",
-      score: 0.9991282820701599
-    },
-    {
-      displayName: "ice cream, icecream",
-      score :0.00010875583393499255
-    },
-    {
-      displayName: "thimble",
-      score: 0.00008958124089986086
-    }
-  ];
-  /////////////////
+  const classifierResults = await service(args);
```

Comment:
thank you!

Reply:
I was freaking out how it was finding Strawberry in everything, trying to figure out Mobilenet's fruit bias, until I finally stepped through Classifier.js.

Reply:
🤣  ... I do (now) remember you telling me (earlier) that it was broken, for the record.